src/gtimelog/timelog.py

```diff
@@ -1107,21 +1107,23 @@ def check_reload(self):
     def load(self):
         """Load task list from a file named self.filename."""
         groups = {}
+        others = []
         self.last_mtime = get_mtime(self.filename)
         try:
             with open(self.filename, encoding='utf-8') as f:
                 for line in f:
                     line = line.strip()
                     if not line or line.startswith('#'):
                         continue
-                    if ':' in line:
+                    if ':' in line:  # tasks with group prefix
                         group, task = [s.strip() for s in line.split(':', 1)]
-                    else:
-                        group, task = self.other_title, line
-                    groups.setdefault(group, []).append(task)
+                        groups.setdefault(group, []).append(task)
+                    else:  # "other" tasks
+                        others.append(line.strip())
         except IOError:
             pass # the file's not there, so what?
-        self.groups = sorted(groups.items())
+        # append the "other" tasks at the end
+        self.groups = list(groups.items()) + [(self.other_title, others)]
```

Comment:
What if there are no tasks in the "others" group?  I suggest
```suggestion
        self.groups = list(groups.items())
        if others:
            self.groups.append((self.other_title, others))
```

Reply:
Yes, well spotted. Thanks!